ogre/src/OgreCamera.cc

```diff
@@ -66,6 +66,14 @@ void OgreCamera::Destroy()
   }
 }
 
+//////////////////////////////////////////////////
+math::Angle OgreCamera::HFOV() const
+{
+  double vfov = this->ogreCamera->getFOVy().valueRadians();
+  double hFOV = 2.0 * atan(tan(vfov / 2.0) * this->AspectRatio());
```

Comment:
nit (to align with `vfov` which is used throughout):
```suggestion
  double hfov = 2.0 * atan(tan(vfov / 2.0) * this->AspectRatio());
```

Same for `Ogre2Camera::HFOV`

Reply:
Not using lowercase because there is a "member variable with the same name" compiler warning.